examples/asyncCall.ts

```diff
@@ -40,40 +41,31 @@ const walletAddress = await wallet.getAddressHex();
 // biome-ignore lint/nursery/noConsole: <explanation>
 console.log("walletAddress", walletAddress);
 
-await faucet.withdrawTo(walletAddress, 100000n);
+const faucetHash = await faucet.withdrawTo(walletAddress, 100_000_000n);
 
-while (true) {
-  const balance = await client.getBalance(walletAddress, "latest");
-  if (balance > 0) {
-    break;
-  }
-}
+await waitTillCompleted(client, 1, bytesToHex(faucetHash));
 await wallet.selfDeploy(true);
 
 // biome-ignore lint/nursery/noConsole: <explanation>
 console.log("Wallet deployed successfully");
 
 const anotherAddress = WalletV1.calculateWalletAddress({
   pubKey: pubkey,
-  shardId: 1,
+  shardId: 2,
   salt: 200n,
 });
 
-await wallet.sendMessage({
+const hash = await wallet.sendMessage({
   to: anotherAddress,
-  value: 10n,
-  gas: 100000n,
+  value: 10_000_000n,
```

Comment:
Why do we need another value?

Reply:
because for async call it should cover gas. it should be now at least 10*gas in value to be sent